lib/new-admin/graphql/types/index.js

```diff
@@ -18,6 +18,7 @@ const status = require('./status.type')
 const transaction = require('./transaction.type')
 const user = require('./users.type')
 const version = require('./version.type')
+const customInfoRequests = require('./customInfoRequests.type')
```

Comment:
New imports could maintain the alphabetical order in place. This also happens on `lib/new-admin/graphql/resolvers/index.js`